python/pyspark/pandas/namespace.py

```diff
@@ -2480,6 +2531,7 @@ def isna(obj):
 isnull = isna
 
 
+@no_type_check
```

Comment:
Out of curiosity, what methods should we mark as `no_type_check`? With only one parameter that is of Any type?

Reply:
According to its definition, `no_type_check` is `Decorator to indicate that annotations are not type hints.`. 
Are there any annotations in this method?

Reply:
These could be used as a udf for `transform` or other functions, although usually it shouldn't happen.
In that case, it has to know the return type but if there is a return type annotation, it can't infer the return type.

Reply:
Makes sense! Thanks.